x/genutil/config/priv_validator_key.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
err, what's this file?

Reply:
ugh, I've been switching frequently between this repo and the SDK repo, likely merged by accident

will ve removed